crates/bevy_text/src/pipeline.rs

```diff
@@ -146,7 +153,7 @@ impl TextPipeline {
             spans.push((span_index, span, text_font, face_info, color));
         }
 
-        let line_height = font_size * 1.2;
+        let line_height = line_height_pixel.max(line_height_scalar * font_size);
```

Comment:
Hmm... If I understand it correctly, this should produce a combination of the largest font size with the largest line height scalar, even if they weren't used on the same span. This can produce unexpected line height results if you have a small font with a large scalar and a large font with a small scalar on the same line.